yt_idv/shader_objects.py

```diff
@@ -243,6 +243,7 @@ class Shader(traitlets.HasTraits):
     """
 
     _shader = None
+    allow_null = traitlets.Bool(True)
```

Comment:
IMO keeping this at True is the way to go so that the current behavior is default -- don't want shader compilation errors to crash the GUI, and debugging or re-compiling new shaders on the fly is nicer when the exception is caught. But adding this attr allows a convenient switch to flip during testing.

Reply:
I think that's a good point.  We could also expose it per-shader-using object as a traitlet, and then have that be passed down.  That may be overkill though.